source-academy/cadet-frontend
# Fix playground hash links

### Description

Fix playground hash links.

Fixes #1472.

### Type of change


### How to test

Change your language to something other than Source 4 with no library, then try this link: http://localhost:8000/playground#chap=4&prgrm=PTAEGUEkGEAVQFLlARgHQCZMChsDMBXAOwGMAXASwHsjQ8KAjACiIEpQBvbUH0AJwCmZAn1q0AvJNAAGbr3m8A-DLkL5ALlASpKVWvnLd+tZvrNaAWlTsA1HUYtQVjKwDc2AL64zTAGxugA

It should reliably change the language to Source 4 with no library.

### Checklist
